.github/workflows/github-actions.yml

```
@@ -0,0 +1,41 @@
+name: GitHub Actions
+on: [push]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-20.04
```

```suggestion
    runs-on: ubuntu-latest
```

Should be better to test on the latest version of Ubuntu.